_search-plugins/searching-data/search-shard-routing.md

```diff
@@ -0,0 +1,212 @@
+---
+layout: default
+parent: Searching data
+title: Search shard routing
+nav_order: 70
+---
+
+# Search shard routing
+
+To ensure redundancy and improve search performance, OpenSearch distributes index data across multiple primary shards, with each primary shard having one or more replica shards. When a search query is executed, OpenSearch routes the request to a node containing either primary or replica shard of the index. This technique is known as _search shard routing_.
+
+
+## Adaptive replica selection
+
+In OpenSearch, in order to improve latency, search requests are routed using _adaptive replica selection_, which chooses the nodes based on the following factors:
+
+- The time it took a particular node to run previous requests.
+- The latency between the coordinating node and selected node.
+- The size of the queue of the node's search thread pool.
+
+If you have permissions to call the OpenSearch REST APIs, you can turn off search shard routing. For more information about the REST API user access, see [REST management API settings]({{site.url}}{{site.baseurl}}/install-and-configure/configuring-opensearch/security-settings/#rest-management-api-settings). To disable search shard routing, update the cluster settings as follows:
+
+```json
+PUT /_cluster/settings
+{
+  "persistent": {
+    "cluster.routing.use_adaptive_replica_selection": false
+  }
+}
+```
+{% include copy-curl.html %}
+
+If you turn off search shard routing, OpenSearch will use round-robin routing, which can negatively impact search latency.
+{: .note}
+
+## Node and shard selection during searches
+
+OpenSearch uses all nodes to choose the best routing for search requests. However, there might be reasons why you want to manually select the nodes or shards to which the search request is sent. Some of these reasons are:
+
+- Using cached previous searches.
+- Dedicating specific hardware for searches.
+- Using only local nodes for searches.
+
+You can use the `preference` parameter in the search query to indicate the destination for the search. The following is the complete list of available options:
+
+1. `_primary`: Forces the search to execute only on primary shards.
```

Comment:
Should each of these end in a colon?

Reply:
I don't believe so, unless we rework this part entirely, not to have colon and colon in the same line.

Reply:
@AntonEliatra That's a fair point, and I realize we're using a slightly different structure here than usual, so either way is fine with me 😄.